### PR TITLE
MPI3SNP

### PR DESCRIPTION
**Pull request by branfosj**
For INC1072512

* [x] Assigned to reviewer

`MPI3SNP-f03aecf-foss-2019b.eb`
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] EL7-power9
* [ ] Ubuntu16 VM

`MPI3SNP-f03aecf-fosscuda-2019b.eb`
* [ ] EL7-haswell
* [ ] EL7-power9

### CODE REVIEW

File: easybuild/easyconfigs/m/MPI3SNP/MPI3SNP-f03aecf-fosscuda-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+dependencies = []
+
+configopts = '-DTARGET_ARCH=GPU -DCMAKE_CXX_FLAGS="$CMAKE_CXX_FLAGS -pthread" '
+
+import subprocess as _subprocess  # NOQA
+_arch = _subprocess.check_output(['uname', '-m'], universal_newlines=True).strip()
+if _arch == 'ppc64le':
+    patches = ['%(name)s-%(version)s_remove_nvcc_flag.patch']
+    checksums.append('f73c0d90488b36fcae92c521af4ec625a2b85d5ed609acf762017f5bf2d3be31')
+    configopts += '-DCMAKE_CUDA_FLAGS="-arch=sm_70 -Xcompiler -mno-float128 " '
```
Comment (edmondac) [inline easybuild/easyconfigs/m/MPI3SNP/MPI3SNP-f03aecf-fosscuda-2019b.eb:29]: Do you also want to set `-arch` for haswell GPU nodes?

Comment (branfosj) [reply easybuild/easyconfigs/m/MPI3SNP/MPI3SNP-f03aecf-fosscuda-2019b.eb:32]: fixed in #f092760

**Review by edmondac — APPROVED** [review]
lgtm

**Comment by edmondac** [comment]
See https://github.com/bear-rsg/easybuild-easyconfigs/pull/324/commits/92b3ad053f8597fe57990264b04c180c42542b07 for a checksum fix

**Pull request merged**